specs/vep-554-apache-airflow-integration/README.MD

```diff
@@ -0,0 +1,401 @@
+# VEP-554: Apache Airflow Integration
+
+* **Author(s):** Miroslav Ivanov (miroslavi@vmware.com), Antoni Ivanov (
+  aivanov@vmware.com)
+* **Status:** draft
+
+To get started with this template:
+
+- [ ] **Create an issue in Github (if one does not exists already)**
+- [ ] **Make a copy of this template directory.**
+  Copy this template into the specs directory and name it
+  `NNNN-short-descriptive-title`, where `NNNN` is the issue number (with no leading-zero
+  padding) created above.
+- [ ] **Fill out this file as best you can.**
+  There are instructions as HTML comments. At minimum, you should fill in the "Summary"
+  and "Motivation" sections.
+- [ ] **Create a PR for this VEP.**
+- [ ] **Merge early and iterate.**
+  Avoid getting hung up on specific details and instead aim to get the goals of the VEP
+  clarified and merged quickly. The best way to do this is to just start with the
+  high-level sections and fill out details incrementally in subsequent PRs.
+
+- [Summary](#summary)
+- [Glossary](#glossary)
+- [Motivation](#motivation)
+- [Requirements and Goals](#requirements-and-goals)
+    - [Goals](#goals)
+    - [Non-goals](#non-goals)
+- [High-level design](#high-level-design)
+- [System Interface (API)](#api-design)
+- [Detailed design](#detailed-design)
+    - [VDK Provider](#vdk-provider)
+        - [VDK Hook](#vdk-hook)
+        - [VDK Operator](#vdk-operator)
+        - [VDK Sensor Operator](#vdk-sensor-operator)
+        - [Connection](#connection)
+        - [Sample DAG](#sample-dag)
+- [Security and Permissions](#security-and-permissions)
+- [Implementation Stories](#implementation-stories)
+- [Alternatives](#alternatives)
+
+## Summary
+
+This VEP outlines the architectural changes required to provide VDK users with the
+ability to do multi-jobs non-linear analytics. The support for definition of dependencies
+between units of work (tasks or jobs) will be introduced by integrating VDK with Apache
+Airflow.
+
+The integration itself will be achieved through the VDK Provider implementation. The
+provider will be deployed on private Airflow deployment and will be responsible for
+managing Data Jobs.
+
+## Glossary
+
+* **VDK**: Versatile Data Kit
+* **Control Service**: VDK Control Service
+* **Data Job**: VDK Data Job
```

Comment:
We have a dictionary - https://github.com/vmware/versatile-data-kit/wiki/dictionary 

So you can simply link the dictionary for some common terms which are already defined there and do not to re-define them there.

Reply:
Thank you. I was looking for this page but I wasn't able to find it.